app/src/main/java/com/nervousfish/nervousfish/activities/KeyManagementActivity.java

```diff
@@ -17,6 +45,67 @@
     @Override
     protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_key_management);
+        this.setContentView(R.layout.activity_key_management);
+        final IServiceLocator serviceLocator = NervousFish.getServiceLocator();
+        final IDatabase database = serviceLocator.getDatabase();
```

Comment:
I mean that this is good practice, to store the database in a temporary variable before you use it :+1 You haven't always done that